wait/host_port.go

```diff
@@ -193,7 +203,7 @@ func externalCheck(ctx context.Context, ipAddress string, port nat.Port, target
 	}
 }
 
-func internalCheck(ctx context.Context, internalPort nat.Port, target StrategyTarget) error {
+func InternalCheck(ctx context.Context, internalPort nat.Port, target StrategyTarget) error {
```

Comment:
bug: this shouldn’t be exported it’s a private method

Reply:
This is needed to avoid exporting `mockStrategyTarget `. I've explained it better here: https://github.com/testcontainers/testcontainers-go/pull/2778#discussion_r1760575891

Reply:
Sorry we can't export an internal function just be support testing.

Reply:
done.